app/views/chargeback/_cb_assignments.html.haml

```diff
@@ -27,36 +26,22 @@
         :javascript
           miqInitSelectPicker();
           miqSelectPickerEvent("cbtag_cat", "#{url}")
-
   - unless @edit[:new][:cbshow_typ].nil? || @edit[:new][:cbshow_typ] == "nil"
     - if !@edit[:new][:cbshow_typ].ends_with?("-tags") || @edit[:new][:cbtag_cat].present?
       %hr
       %h3
         = _('Selections')
-      %table.table.table-bordered.table-striped
-        %thead
-          %tr
-            %th= _('Name')
-            %th= _('Rate')
-        %tbody
-          - if @edit[:new][:cbshow_typ].ends_with?("-tags")
-            - @edit[:cb_assign][:tags].invert.sort_by { |a| a.first.downcase }.each do |tag, id|
-              %tr
-                %td
-                  = h(tag)
-                %td
-                  - options = @edit[:cb_rates].invert.sort
-                  = select_tag("#{@edit[:new][:cbshow_typ]}__#{id}",
-                              options_for_select([[nothing, "nil"]] + options, @edit[:new]["#{@edit[:new][:cbshow_typ]}__#{id}".to_sym].to_s),
-                              "data-miq_sparkle_on" => true, "data-miq_sparkle_off" => true, :class    => "selectpicker")
-                :javascript
-                  miqInitSelectPicker();
-                  miqSelectPickerEvent("#{@edit[:new][:cbshow_typ]}__#{id}", "#{url}")
-          - else
-            - @edit[:cb_assign][:cis].invert.sort_by { |a| a.first.downcase }.each do |ci, id|
-              %tr#new_tr
-                %td
-                  = h(ci)
+      - if @edit[:new][:cbshow_typ] == "tenant"
+        %table.table.table-bordered.table-hover.table-treegrid
+          %thead
+            %tr
+              %th= _('Name')
+              %th= _('Rate')
+          %tbody
+            - @edit[:cb_assign][:herarchy].sort_by! { |a| a.first }.each do |id, data|
+              %tr{:'id' => 'tenant_' + id.to_s, :'class' => 'collapsed'}
```

Comment:
You can use `:id => 'tenanat_...'` until there is no special symbol in the hash key!